LDSContent/ContentController.swift

```
@@ -217,8 +217,11 @@ extension ContentController {
             }
         }
         let mergedURL = URL(fileURLWithPath: mergedPath)
+        
         let directory = mergedURL.deletingLastPathComponent()
-        try FileManager.default.createDirectory(at: directory, withIntermediateDirectories: true, attributes: nil)
+        if !FileManager.default.fileExists(atPath: directory.path) {
```

In general it's a safer pattern to attempt to create a directory, and only if that fails check to see if it exists. If it exists, you can ignore the error. If not, then you can fail the operation.